homeassistant/components/sonos/speaker.py

```diff
@@ -502,15 +502,18 @@ async def async_unseen(self, now: datetime.datetime | None = None) -> None:
             self._seen_timer()
             self._seen_timer = None
 
-        hostname = uid_to_short_hostname(self.soco.uid)
-        zcname = f"{hostname}.{MDNS_SERVICE}"
-        aiozeroconf = await zeroconf.async_get_async_instance(self.hass)
-        if await aiozeroconf.async_get_service_info(MDNS_SERVICE, zcname):
-            # We can still see the speaker via zeroconf check again later.
-            self._seen_timer = self.hass.helpers.event.async_call_later(
-                SEEN_EXPIRE_TIME.total_seconds(), self.async_unseen
-            )
-            return
+        if now:
```

Comment:
What do you think about renaming this parameter to `callback_triggered_at` or something? `now` isn't a very helpful name. I presume it comes from the callback mechanism calling this method with the timestamp of right now.